src/views/forms/FormSubWalletCreation/FormSubWalletCreationTs.ts

```diff
@@ -306,7 +334,100 @@ export class FormSubWalletCreationTs extends Vue {
       this.networkType,
       childWalletName,
     )
-
     return wallet
+    }   
+  }
+
+  async importSubAccountFromLedger(childWalletName: string) {
+    const subWalletName = childWalletName
+    const accountPath = this.currentWallet.values.get("path")
+    const currentAccountIndex = accountPath.substring(accountPath.length-2,accountPath.length-1)
```

Comment:
symbol uses hardened paths but you should not read the index like this ... please use `DerivationPathService`.